src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/MakeAnApplicationTypesProvider.java

```diff
@@ -40,8 +40,7 @@ public DynamicList getMakeAnApplicationTypes(Callback<AsylumCase> callback) {
 
         boolean hasHomeOfficeRole = userDetails.getRoles()
             .stream().anyMatch(HO_ROLES::contains);
-        boolean isInternalAdaAndAdminRole = hasRole(ROLE_ADMIN)
-            && isInternalCase(asylumCase) && isAcceleratedDetainedAppeal(asylumCase);
+        boolean isInternalAndAdminRole = hasRole(ROLE_ADMIN) && isInternalCase(asylumCase);
```

Comment:
Do we need to update unit test? With ADA removed

Reply:
Updated in latest commit.